src/tests/ftest/dfuse/mu_perms.py

```diff
@@ -365,9 +367,9 @@ def _verify(use_il, expected_il_messages, expect_der_no_perm):
             # which the user also doesn't have
             verify_perms_cmd.update_params(perms=posix_perms)
             self.log.info('Verify - no perms - not using IL')
-            _verify(use_il=False, expected_il_messages=0, expect_der_no_perm=False)
+            _verify(use_il=False, expected_il_messages=0, expect_der_no_perm=False, il_lib=il_lib)
```

Comment:
... and then we don't need to pass these either
```suggestion
            _verify(use_il=False, expected_il_messages=0, expect_der_no_perm=False)
```

Reply:
Thank you! Fixed as you suggested.